### PR TITLE
Supported file:// url for tuftool download

### PR DESCRIPTION
**Pull request by srgothi92**
*Issue #, if available:*
#222 

*Description of changes:*
* Updated tuftool README.md with an example of `tuftool downlaod`
* Added test to download using file:// form local repository.

Testing:
* Followed updated tuftool README.md to create and download tuf repo
* Ran the added test case to download form local repository.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by zmrow — APPROVED** [review]
:yum:

File: tuftool/README.md [hunk 1]
```diff
@@ -109,6 +109,14 @@ tuftool update \
    --timestamp-version 2 \
    --outdir "${WRK}/tuf-repo" \
    --metadata-url file:///$WRK/tuf-repo/metadata
+
+# finally, downlaod tuf repo
```
Comment (webern) [inline tuftool/README.md]: Per discussion elsewhere, this example could be given in another section. Or, if not, the comment should at least explain that the download command is usually used to download a remote repo with an HTTP/S URl.

**Review by webern — COMMENTED** [review]
Looks good except that the README might cause confusion (i.e. 'why download a local repo?')

**Comment by srgothi92** [comment]
Addressed Readme comments.

**Review by webern — COMMENTED** [review]
.

**Review by webern — APPROVED** [review]
:lobster:

**Comment by srgothi92** [comment]
Found a better way to create "file" based URL.

**Pull request merged**